cmd/infra/powervs/create.go

```diff
@@ -1366,3 +1397,18 @@ func (infra *Infra) isCloudConnectionReady(ctx context.Context, options *CreateI
 	log(infra.ID).Info("PowerVS Cloud Connection ready")
 	return nil
 }
+
+// attachTag attach tags to cloud resources
```

Comment:
can you please add more information like tag format which it adds to the resources?

Reply:
Added.